monitor.js

```diff
@@ -2,15 +2,24 @@
  * Monitor
  * Pokemon Showdown - http://pokemonshowdown.com/
  *
- * Various utility functions to make sure PS is running healthy.
+ * Various utility functions to make sure PS is running healthily.
  *
  * @license MIT license
  */
 'use strict';
 
 const FS = require('./fs');
 
+/**
+ * @class TimedCounter
```

Comment:
I thought it was necessary for `tsc` to recognize `TimedCounter` as a type, but it turns out not to be. I could use it as a place to document why the class exists while I add documentation for `TimedCounter#increment`